src/pages/ExamsPage/components/AttemptList.jsx

```diff
@@ -1,9 +1,143 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, DataTable } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
 
-const AttemptList = () => (
-  <div data-testid="attempt_list">
-    Placeholder for attempt list
-  </div>
+const capitalizeFirstLetter = (string) => string.charAt(0).toUpperCase() + string.slice(1);
+
+const ResetAction = (props) => (
+  <Button onClick={() => console.log('Reset', props)}>
+    Reset
+  </Button>
 );
 
+const AttemptList = ({ attempts }) => {
+  const { formatMessage, formatDate } = useIntl();
+
+  return (
+    <div data-testid="attempt_list">
+      <p>Data Table</p>
+      <DataTable
+        isPaginated
+        isSelectable
+        initialState={{
+          pageSize: 20,
+        }}
+        isSortable
+        itemCount={attempts.length}
+        bulkActions={[
+          <ResetAction />,
+        ]}
+        additionalColumns={[
+          {
+            id: 'action',
+            Header: formatMessage({
+              id: 'AttemptsList.action',
+              defaultMessage: 'Action',
+              description: 'Table header for the table column listing action to reset the exam attempt',
+            }),
+            // TODO: For MST-1945, make this button actually "reset" (i.e. delete)
+            // the exam attempt for the cell's respective row after confirmation.
+            /* eslint-disable react/prop-types, react/no-unstable-nested-components */
+            Cell: ({ row }) => <Button variant="link" size="sm" onClick={() => console.log(`Resetting ${row.values.name}`)}>Reset</Button>,
```

Comment:
`<ResetAction>` is a "bulk action", meaning it can be performed on multiple rows if selected. I've added a comment + renamed it to BulkResetAction to clarify this.

Reply:
does this mean you can reset multiple rows at a time? How would this work with the modal that pops up to confirm deletion?

Reply:
Yes, you can reset multiple rows at a time (see: [docs](https://paragon-openedx.netlify.app/components/datatable/#bulk-actions)). Since the action is just a component that's passed in, I think you could theoretically insert a modal into this component and only perform the action when the "yes" button is pressed, though this would have to be tested.

Reply:
is there a way to just turn the bulk feature off? Seems like extra effort to build and test this. I don't think instructors typically need to do large scale resetting.

Reply:
Easy enough, pushing the change now.